uint16 current
int16 pwr
int16 vel
Encoder enc
